### PR TITLE
knativeproject kiosk annoucement KubeconEU23

### PR DESCRIPTION
**Pull request by nainaz**
"Fixes #1297" or "Add description of the problem this PR solves"

## Proposed Changes 

-
-
-

### CODE REVIEW

File: blog/docs/events/knative-projectmeeting-kubecon-eu-2023.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+During this event, you'll have the opportunity to engage with Knative experts as they share their insights on the latest advancements and use cases. You'll get a chance to network with peers, ask questions, and  help define Knative's roadmap and community growth.
+Whether you're a seasoned developer or new to cloud-native computing, this meeting is the perfect opportunity to share your use cases and expectations, and expand your knowledge. Mark your calendars and join us for an informative and engaging 2 hours that get you ready to tackle your next Kubernetes project with the Serverless flair!
+P.S: This event is free for all registered attendees for the conference.
+
+## Knative Kiosk
+Knative Project will have a half day kiosk in the Expo area during the Conference. We are looking forward to seeing you in Amsterdam at the Project meeting and at the Knative kiosk in the expo area.
+If you cannot attend the KubeConEU 23, don't worry, you can join the conversations in the #knative channel at CNCF Slack. Say Hi to us today on the CNCF Slack [CNCF Slack] (https://communityinviter.com/apps/cloud-native/cncf)
```
Comment (psschwei) [inline blog/docs/events/knative-projectmeeting-kubecon-eu-2023.md:15]:
```suggestion
If you cannot attend the KubeConEU 23, don't worry, you can join the conversations in the #knative channel at CNCF Slack. Say Hi to us today on the CNCF Slack [CNCF Slack](https://communityinviter.com/apps/cloud-native/cncf)

```

**Review by psschwei — COMMENTED** [review]
:tada: 

I suggest we update https://github.com/knative/docs/blob/main/blog/docs/index.md?plain=1#L25-L29 and to replace the Detroit post with this one

**Comment by nainaz** [comment]
Done!

**Review by psschwei — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**